packages/react-core/src/components/Alert/AlertToggleExpandButton.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Button, ButtonProps, ButtonVariant } from '../Button';
+import { AlertContext } from './AlertContext';
+import AngleRightIcon from '@patternfly/react-icons/dist/esm/icons/angle-right-icon';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/Alert/alert';
+
+export interface AlertToggleExpandButtonProps extends ButtonProps {
+  /** Aria label for the toggle button */
+  'aria-label'?: string;
+  /** A callback for when the toggle button is clicked */
+  onToggleExpand?: () => void;
+  /** Flag to indicate if the content is expanded */
+  isExpanded?: boolean;
+  /** Variant label for the toggle button */
+  variantLabel?: string;
+}
+
+export const AlertToggleExpandButton: React.FunctionComponent<AlertToggleExpandButtonProps> = ({
+  'aria-label': ariaLabel,
+  variantLabel,
+  onToggleExpand,
+  isExpanded,
+  ...props
+}: AlertToggleExpandButtonProps) => {
+  const { title, variantLabel: alertVariantLabel } = React.useContext(AlertContext);
+  return (
+    <Button
+      variant={ButtonVariant.plain}
+      onClick={onToggleExpand}
+      aria-expanded={isExpanded}
+      aria-label={ariaLabel === '' ? `Toggle ${variantLabel || alertVariantLabel} alert: ${title}` : ariaLabel}
+      aria-labeledby={`${title} toggle`}
```

Comment:
fyi this attr is spelled `aria-labelledby`, and it takes the value of an `id` attribute of an element that labels this element (it can take multiple `id` values, if multiple elements label it). The value shouldn't be a string though. An `aria-label` may be enough here, or just whatever's consistent with where we're doing this in other places.

Reply:
Thanks, I was trying to get parity with core, but in this case we are not even exposing that `id`  to be customizable for the user, so I will just remove it.